.gitignore

```diff
@@ -6,6 +6,7 @@
 *.user
 *.userosscache
 *.sln.docstates
+*.project.lock
```

Comment:
need to append with ".json". I think 14.8k of your 15.8 k lines of changes are just that one file, and its still here :P